benchmarks/bench.js

```diff
@@ -2,7 +2,7 @@
 
 const Fs = require('fs');
 
-const Assert = require('@hapi/hoek/lib/assert');
+const Assert = require('@hapi/hoek/assert');
```

Comment:
@Marsup this broke builds for us after updating our package-locks.json files. this was installed as a transient dependency of another transient dependency package called `joi`. I have no idea what any of these packages do but this is the error I get when I run unit tests that use the `node-jq` library:
![resim](https://user-images.githubusercontent.com/6615389/219977882-ba9e45ff-c9ae-4687-99b3-3ede5a9a3a6c.png)

Reply:
The constraint on hoek@^11 is correct though, make sure you upgraded your lockfile properly.

Reply:
Oh, wait, you're actually pointing at the benchmarks, why would it even care, it's not even exported by the module.

Reply:
ok so that's my bad sorry I chose the wrong file. it's the ....lib/index.js file as seen in the screen shot. I can see that it was changed too. please check #2911 I also opened an issue